README.md

```diff
@@ -32,10 +32,11 @@ Example config:
 global:
   scrape_interval: 10s
 scrape_configs:
-  - job_name: nrpe
+  - job_name: nrpe_check_load
     metrics_path: /export
     params:
       command: [check_load] # Run the check_load command.
+      ssl: [true]
```

Comment:
Probably worth a comment, not everyone will want this

Reply:
done